src/SDKs/SqlManagement/Management.Sql/Properties/AssemblyInfo.cs

```diff
@@ -22,7 +22,7 @@
 [assembly: AssemblyTitle("Microsoft Azure SQL Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure SQL.")]
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.8.0.0")]
+[assembly: AssemblyFileVersion("1.9.0.0")]
```

Comment:
please revert, 1.8.0 has not yet been published ( see https://www.nuget.org/packages/Microsoft.Azure.Management.Sql )